lua/sig-window-nvim.lua

```diff
@@ -29,7 +36,9 @@ local function highlight_text(bufnr, start_ix, end_ix, highlight_group)
   end
 end
 
-local function calc_window_dimensions(text, max_width, max_height)
+local function calc_window_dimensions(lines, max_width, max_height)
+  text = table.concat(lines, "\n")
```

Comment:
I think this is over-simplifying the problem. Each line should be checked to determine the appropriate width and height. I'm ok with fixing this after merging the PR, I think this simple solution is sufficient for now. Feel free to leave this as is or take a stab at it.